src/_data/certifiedNodes.js

```diff
@@ -0,0 +1,15 @@
+const fetch = require('node-fetch');
```

Comment:
Can we use Eleventy Fetch instead? It does some caching to make the builds somewhat faster

https://www.11ty.dev/docs/plugins/fetch/